src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const smallDiscount = 20;
+  const basicDiscount = 50;
+  const rentalCost = days * dayPrice;
```

Comment:
You should still capitalize those constants, because they should never change in your code, they are like math consts e.g. `Math.PI` , `Math.SQRT2`